src/App.tsx

```diff
@@ -1,27 +1,60 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+export const App: React.FC = () => {
+  const defaultGoods: Good[] = [];
+  const [goods, setGoods] = useState(defaultGoods);
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+  const getAllHere = () => {
+    getAll().then(good => {
+      setGoods(good);
+    });
+  };
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+  const get5Here = () => {
+    get5First().then(good => {
+      setGoods(good);
+    });
```

Comment:
```suggestion
    get5First().then(setGoods);
```
Since `setGoods` accepts good as argument (exactly what your promise returns) you can just pass it as callback into `then`